BearMyBanner/settings/IBMBSettings.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace BearMyBanner.Settings
+{
+    public interface IBMBSettings
```

Comment:
This interface is needed to abstract away ModLib settings, because if included they will require reference during test run, thus causing errors.